analyses/PA_cd_2000/doc_PA_cd_2000.md

```diff
@@ -0,0 +1,18 @@
+# 2000 Pennsylvania Congressional Districts
+
+## Redistricting requirements
+In ``Pennsylvania``, according to [NCSL Redistricting Law 2000](https://web.archive.org/web/20041216185957/https://www.senate.mn/departments/scr/redist/red2000/Tab5appx.htm), there are no state law requirements for congressional districts.
```

Comment:
Can you add back the text here for contiguity and equal population, otherwise this looks great!